specifications/barriers/README.md

```diff
@@ -0,0 +1,17 @@
+# Barrier synchronization
+-
+
+## Barriers.tla
+
+A specification of a Reusable Barrier synchronization facility using as
+presented in the INFO09012 Parallel Programming course in ULiège by 
+Prof. Pascal Fontaine.
```

Comment:
People won't have access to those lecture notes, could you describe the algorithm in words?